mesonbuild/interpreter/compiler.py

```diff
@@ -164,6 +171,8 @@ def stderr_method(self, args: T.List['TYPE_var'], kwargs: 'TYPE_kwargs') -> str:
 _COMPILES_KWS: T.List[KwargInfo] = [_NAME_KW, _ARGS_KW, _DEPENDENCIES_KW, _INCLUDE_DIRS_KW, _NO_BUILTIN_ARGS_KW]
 
 _HEADER_KWS: T.List[KwargInfo] = [REQUIRED_KW.evolve(since='0.50.0', default=False), *_COMMON_KWS]
+_HAS_KWS: T.List[KwargInfo] = [REQUIRED_KW.evolve(since='1.1.0', default=False), *_COMMON_KWS]
+_HAS_ARGUMENT_KWS: T.List[KwargInfo] = [REQUIRED_KW.evolve(since='1.1.0', default=False)]
```

Comment:
Why is this a list?

Why don't we evolve it once and use it plain, but also use it in _HAS_KWS?

Reply:
Indeed, fixed.